### PR TITLE
fix composition of the start command with user params, add a test

### PR DESCRIPTION
**Pull request by ajm01**
Signed-off-by: Andrew J. Mauer <ajmauer@gmail.com>

### CODE REVIEW

File: tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java [hunk 1]
```diff
@@ -269,6 +269,42 @@ public void testStartWithParms() {
         // Close the terminal.
         terminal.close();
     }
+    
+    /**
+     * Tests the start with parameters command string for proper composition.
```
Comment (scottkurz) [inline tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java:274]: I don't think this test is worth adding.  

Maybe it was a bad idea.  

To at least illustrate roughly what I was thinking, consider this.
https://github.com/scottkurz/liberty-tools-eclipse/tree/start-with-params

Note though we have to refactor the bundle by exposing / consuming this util package to test in this fashion.

Let's talk more on our scrum.

**Review by scottkurz — CHANGES_REQUESTED** [review]

**Review by scottkurz — COMMENTED** [review]
Test not really needed

File: tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java (hunk 1 above)
Comment (ajm01) [reply tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java]: this re-working of the order of user parameters does not work with gradle.  if, say, '--hotTests' is placed before the LibertyDev command, gradle fails with an unknown cmd arg failure. Gradle thinks the --hotTests arg applies to gradle itself and not th eLibertyDev call.

Comment (scottkurz) [reply tests/src/liberty/tools/test/it/LibertyPluginSWTBotMavenTest.java]: Let's forget this idea then.  I hadn't thought this through.  We'll go with the existing behavior, update the UFO, and revisit this in:  https://github.com/OpenLiberty/liberty-tools-eclipse/issues/111   for MVP.

**Pull request closed without merging**